### PR TITLE
Replace Hard-Coded Sensible Capacity Curves with ADP/BF

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

[improves the design airflow/design SHR calculation in apply_hvac_equipment_adjustments within hvac_sizing.rb for central-ducted air systems used for cooling]

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [x] ~~Schematron validator (`EPvalidator.xml`) has been updated~~
- [x] ~~Sample files have been added/updated (`openstudio tasks.rb update_hpxmls`)~~
- [x] ~~Tests have been added/updated  (`HPXMLtoOpenStudio/tests/test_hvac_sizing.rb` and/or `workflow/tests/test*.rb`)~~
- [x] Documentation has been updated (no change necessary)
- [x] ~~Changelog has been updated~~
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files. TODO: ~~merge and test w/ diverse set of buildings/climates in ResStock~~ house 237.xml (Florida) edge case was investigated, and the ADP/BF approach for this case produces more reasonable equipment capacities (see attached excel, ppt).

[ADP_BF_ResStock Testing Results.pptx](https://github.com/user-attachments/files/17078608/ADP_BF_ResStock.Testing.Results.pptx)

[testing_ResStock_datapoint_237.xlsx](https://github.com/user-attachments/files/17078489/testing_ResStock_datapoint_237.xlsx)

### CODE REVIEW

File: HPXMLtoOpenStudio/resources/hvac.rb
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/hvac.rb:3340]: 👍

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/hvac_sizing.rb:4991]: When you resolved conflicts, you reverted a bunch of changes here and below that we want. Can you fix this?

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/hvac_sizing.rb:5001]: No need to add this property to the object. Just use a local variable in the `apply_hvac_equipment_adjustments` method (e.g., `design_shr` instead of `hvac_sizings.Design_SHR`).

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/hvac_sizing.rb:3647]: Looks like you reverted a change here too. Please fix.

**Review by shorowit — COMMENTED** [review]
I'll probably clean up the code a bit before merging to remove unnecessary comments, but overall it looks pretty reasonable from a programming perspective. Just a couple comments. Relying on @jonwinkler for the engineering review.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:5001]: @shorowit ok. will update the variable name.

Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:5001]: refactor done, changes committed.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:3647]: @shorowit I made some local corrections to the comments above self.get_nominal_speed, committed them, and performed a git diff between the remote branch adp_bf_method_design_shr and remote master. In VS code terminal this is 

git diff adp_bf_method_design_shr origin/master  

There is no diff for the get_nominal_speed method now. What is in adp_bf branch matches master exactly.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:4991]: I see the diffs, they were just in the comments around the HVACSizingValues class. I resolved them to match master and committed.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (jonwinkler) [inline HPXMLtoOpenStudio/resources/hvac_sizing.rb:2162]: @Sashadf1 Maybe add some comments above this line indicating this is an initial assumption prior to attempting to balance sensible and latent design capacity by applying the over size limits.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (jonwinkler) [inline HPXMLtoOpenStudio/resources/hvac_sizing.rb:2191]: @Sashadf1, Should this be:

`cool_lat_cap_design = [cool_cap_design - cool_sens_cap_design, 1.0].max`

I think `cool_cap_design` may change as the loop iterates, right?

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:2124]: ok, will do.

Can you clarify the line number, and which initial assumption?

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:2153]: yes, cool_cap_design may change as the loop iterates.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (jonwinkler) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb:2126]: I was thinking the line `cool_cap_design = hvac_sizings.Cool_Load_Tot` could use an explanation. I know it's obvious based the code that comes next but I thought additional comments would be worth it.

File: HPXMLtoOpenStudio/resources/hvac_sizing.rb
Comment (jonwinkler) [inline HPXMLtoOpenStudio/resources/hvac_sizing.rb:2187]: Consider this optional, but renaming `hvac_cooling_shr` to something like `hvac_cooling_shar_rated` might be nice for clarity.

Comment (Sashadf1) [reply HPXMLtoOpenStudio/resources/hvac_sizing.rb]: done, clarity rename to hvac_cooling_shr_rated

**Comment by Sashadf1** [comment]
@shorowit to resolve merge conflicts and coordinate w/ residential team to test w/ diverse buildstock run

**Review by shorowit — APPROVED** [review]
We have pounded the tires on this enough. Thanks for the good work here, @Sashadf1. I'll merge this in once the CI comes back happy.

**Pull request merged**
